doc/lib_pid.md

```diff
@@ -20,6 +22,8 @@ description:
     initial tuning parameters.  This is a very simple PID controller
     that makes no attempts to tune its parameters.  You must supply them
     for it.  Call PID_init once before your controlling loop begins.
+  * NOTE: limmin and limmax will not limit the value returned by PID_seek, they will however
```

Comment:
When you renamed these to cMin and cMax up above, you didn't change this description, which is confusing.  There isn't anything called "limmin" and "limmax" anymore.